src/oop/programs_with_multiple_classes/medium/q5/Monitor.java

```diff
@@ -0,0 +1,13 @@
+package oop.programs_with_multiple_classes.medium.q5;
+
+public class Monitor {
+    private int countSize;
```

Comment:
Why is this called countSize?  I feel like that does not make sense. Why don't you have `double size` attribute, and then a `isOn` boolean.

Reply:
Then you can change the method in monitor to switch the value of isOn